nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -222,6 +224,22 @@ contract StakingEscrow is Issuer {
         return stakerInfo[_staker].value;
     }
 
+    /**
+    * @notice Get all flags for the staker
+    */
+    function getFlags(address _staker)
+        external view returns (
+            bool windDown,
+            bool reStake,
+            bool measureWork
+        )
+    {
+        StakerInfo storage info = stakerInfo[_staker];
+        windDown = info.flags.bitSet(WIND_DOWN_INDEX);
+        reStake = !info.flags.bitSet(RE_STAKE_DISABLED_INDEX);
+        measureWork = info.flags.bitSet(MEASURE_WORK_INDEX);
+    }
+
```

Comment:
in that case we need make index as public constant and this will increase contract size (because will be new external function), so this is sort of compromise